### DISCUSSION TITLE
Any possibility to support tvOS?

### DISCUSSION THREAD

**Discussion by sureshsu** [invertase/react-native-firebase#5145] [Ideas] [answered]
Given that react-native version for tvOS is separate from iOS, can this react-native-firebase npm package support tvOS as well?

**Comment by mikehardy** [comment, marked as answer]
It could but would need community support to do so - similar to the policy of the upstream firebase-ios-sdk we depend on:

https://github.com/firebase/firebase-ios-sdk#tvos-macos-watchos-and-catalyst

If you had a sketch of how you would support this (with any needed code reorganization or similar) and proposed PRs to do so, I'd be happy to collaborate and merge them in

**Reply by nhannfq** [reply]
any news about this?

**Reply by mikehardy** [reply]
@nhannfq all work here is in the open, any news (if it existed) would show up in the changelog and/or as comments here.

If you want or need this, you'll need to be prepared to do the work to make it happen - but we'll be happy to collaborate on it!
